serverlessworkflow/modules/ROOT/pages/service-orchestration/orchestration-of-openapi-based-services.adoc

```diff
@@ -123,7 +119,7 @@ To send requests with the REST calls, first you need to define the OpenAPI funct
 +
 Once you have access to the OpenAPI specification files, you can define the functions in the Serverless Workflow. 
 
-You can also use the link:{kogito_sw_examples_url}/serverless-workflow-temperature-conversion[`serverless-workflow-temperature-conversion`] example application as a reference.
+You can also use the link:{kogito_sw_examples_url}/serverless-workflow-temperature-conversion[serverless-workflow-temperature-conversion] example application as a reference.
```

Comment:
For consistency, the example applications are written as code. 
```suggestion
You can also use the link:{kogito_sw_examples_url}/serverless-workflow-temperature-conversion[`serverless-workflow-temperature-conversion`] example application as a reference.
```

Reply:
ok...
@hmanwani-rh the main reason I wanted to change this is that in the documentation, with this formatting, we don't make the distinction between a example application mention and a link ...

- `link:{kogito_sw_examples_url}/serverless-workflow-temperature-conversion[`serverless-workflow-temperature-conversion`]`
- `serverless-workflow-temperature-conversion`

both appears as same formatting in the documentation and we may not know that we can click on it ...

Reply:
+1 on the formatting. 
Maybe we can change the style after DP1 to show the code links with an underline. 

CC @ricardozanini

Reply:
do you need an issue for that ?

Reply:
Sure, we can create one.